fs_mark/timing_data.py

```diff
@@ -0,0 +1,109 @@
+# Timing ranges used in fs_mark harness
+
+# For now, use values that always pass
```

Comment:
Will these values ever be changed, or does it make sense to divide them in this way if every target, for each value, assumes the same thing?

Reply:
At this moment these values represent wide enough range to always pass for all targets, later they will be changed for every target accordingly. There is a comment at line 3 about that.